### PR TITLE
Add icarGroupFeedingEventResource

### LINKED ISSUE #382: Group feed utilisation/allowance event
Now that we can support group events, it would be useful to have a group feed utilisation event analogous to the individual animal feed utilisation event.

### PR DESCRIPTION
**Pull request by cookeac**
Add a resource for group feeding or feed allowance. Resolves #382

### CODE REVIEW

**Review by AlexeyHardCode — APPROVED** [review]

File: url-schemes/feedURLscheme.json [hunk 1]
```diff
@@ -714,6 +714,111 @@
...
+                        "$ref": "#/components/parameters/location-id"
+                    }
+                ],
+                "requestBody": {
+                    "required": true,
+                    "description": "The group feeding event to create. \nA client MAY fill in the *Id* field with a client-generated UUID and the server MAY use this *Id*.\nIf the server does not use the client-specified *ID* field it shall issue its own *ID* for the resource.\nA client SHALL ensure that the *meta.source* and *meta.sourceId* fields are filled by the client.\n",
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "$ref": "#/components/schemas/icarFeedIntakeEventResource"
```
Comment (AndreasSchultzGEA) [inline url-schemes/feedURLscheme.json]: icarFeedIntakeEventResource is provides, but response (200) is #/components/schemas/icarGroupFeedingEventResource?

File: resources/icarGroupFeedingEventResource.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+    "description": "Event for recording group or mob feeding. Allowances represent averages so this cannot be used to populate individual animal events.",
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "../resources/icarGroupEventCoreResource.json"
+        }
+    ],
+    "properties": {
+        "feedingEndingDateTime": {
```
Comment (AndreasSchultzGEA) [inline resources/icarGroupFeedingEventResource.json]: feedingEnd**ing**DateTime
Do we use somewhere else the "ing"-naming-scheme?

Comment (cookeac) [inline resources/icarGroupFeedingEventResource.json]:
```suggestion
        "feedingEndDateTime": {

```

File: url-schemes/feedURLscheme.json (hunk 1 above)
Comment (cookeac) [inline url-schemes/feedURLscheme.json]:
```suggestion
                                "$ref": "#/components/schemas/icarGroupFeedingEventResource"
```

**Comment by dirbSEGES** [comment]
Hi @cookeac , will this resource be available in the next release?

**Comment by cookeac** [comment]
Yes, that's right

**Pull request merged**